internal/app/machined/main.go

```diff
@@ -251,8 +251,13 @@ func run() error {
 		return err
 	}
 
-	// Start the machine API.
-	system.Services(c.Runtime()).LoadAndStart(&services.Machined{Controller: c})
+	go func() {
```

Comment:
we don't need a goroutine here, `Start` is non-blocking, it just schedules the service to be loaded